x-pack/plugins/apm/public/components/alerting/ui_components/alert_details_app_section/alert_details_app_section.tsx

```diff
@@ -289,9 +300,32 @@ export function AlertDetailsAppSection({
       }),
     },
   ];
-
   /* Error Rate */
 
+  const getLatencyChartAdditionalData = () => {
+    if (isLatencyThresholdRuleType(alert.fields[ALERT_RULE_TYPE_ID])) {
+      return [
+        <AlertThresholdRect
+          key={'alertThresholdRect'}
+          threshold={alert.fields[ALERT_EVALUATION_THRESHOLD]}
+          alertStarted={alert.start}
```

Comment:
I'm inclined to expose the internals just to make this part more clear:

```suggestion
          y0={alert.fields[ALERT_EVALUATION_THRESHOLD]}
          x1={alert.start}
```

Btw. what is x0 and y1?

Reply:
> Btw. what is `x0` and `y1`?

I found the answer in the docs:

> Each coordinate value can be omitted, if omitted then the corresponding min or max value is used instead.